<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>        
    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Vivian  Brandenburg | Curriculum Vitae</title>
    <meta name="author" content="Vivian  Brandenburg" />
    <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design.
" />


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous" />

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

    <!-- Styles -->
    <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>ðŸ§¬</text></svg>">
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://vivianbrandenburg.github.io/CV_academic/">

    <!-- Dark Mode -->
    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script><!-- google -->
    <meta name="google-site-verification" content="nPIY7yz_3yXwImUhSsPpRGhL2u4Fi5XV5S4Ot9bgtDI" />

  </head>

  <!-- Body -->
  <body class="fixed-top-nav">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="https://vivianbrandenburg.github.io/"><b>Vivian</b> Brandenburg</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">about</a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item dropdown ">
                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">CV</a>
                <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
                  <a class="dropdown-item" href="/CV_academic/">academic CV</a>
                  <div class="dropdown-divider"></div>
                  <a class="dropdown-item" href="/assets/one_page_CV.pdf">one-page CV</a>
                </div>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/contributions/">contributions</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/projects/">projects</a>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/publications/">publications</a>
              </li>

              <!-- Toogle theme mode -->
              <div class="toggle-container">
                <a id="light-toggle">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </a>
              </div>
            </ul>
          </div>
        </div>
      </nav>
    </header>

    <!-- Content -->
    <div class="container mt-5">
      <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">Curriculum Vitae</h1>
            <p class="post-description"></p>
          </header>

          <article>
            <ul id="toc" class="section-nav">
<li class="toc-entry toc-h3"><a href="#professional-career">Professional Career</a></li>
<li class="toc-entry toc-h3"><a href="#education">Education</a></li>
<li class="toc-entry toc-h3"><a href="#publications">Publications</a></li>
<li class="toc-entry toc-h3"><a href="#talks">Talks</a></li>
<li class="toc-entry toc-h3"><a href="#my-subsection">Posters</a></li>
<li class="toc-entry toc-h3"><a href="#teaching">Teaching</a></li>
</ul>
<h3 id="professional-career">
<a class="anchor" href="#professional-career" aria-hidden="true"><span class="octicon octicon-link"></span></a>Professional Career</h3>
<hr>

<table>
  <thead>
    <tr>
      <th>â€ƒ    â€ƒ  â€ƒ  â€ƒ  â€ƒ  â€ƒ â€ƒ â€‰</th>
      <th>â€ƒ</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>since 2025</td>
      <td>
<strong>Scientist Computational Biology</strong>, Resolve Biosciences</td>
    </tr>
    <tr>
      <td>Â </td>
      <td>System Integration</td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2023 â€“ 2024</td>
      <td>
<strong>Postdoctoral Researcher</strong>, Ruhr University Bochum</td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Bioinformatics Group</td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2019 â€“ 2023</td>
      <td>
<strong>Doctoral Researcher</strong>, Ruhr University Bochum</td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Research Groups: Microbial Biology | Bioinformatics</td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2018 â€“ 2019</td>
      <td>
<strong>Teaching Assistant</strong>, Ruhr University Bochum</td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Research Group: Theoretical Computer Science</td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2014</td>
      <td>
<strong>Teaching Assistant</strong>, Leipzig University</td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Research Group: Plant Physiology</td>
    </tr>
  </tbody>
</table>

<p><br></p>
<h3 id="education">
<a class="anchor" href="#education" aria-hidden="true"><span class="octicon octicon-link"></span></a>Education</h3>
<hr>

<table>
  <thead>
    <tr>
      <th>â€ƒ    â€ƒ  â€ƒ  â€ƒ  â€ƒ  â€ƒ â€ƒ   â€ƒ â€ƒ</th>
      <th>â€ƒ</th>
      <th>Â </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2023</td>
      <td>
<strong>Doctoral Degree</strong>, Ruhr University Bochum</td>
      <td>Â </td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Thesis: <em>Investigation of functional RNA structures using deep learning and experiment-guided computational approaches</em>
</td>
      <td>Â </td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2017 â€“ 2019</td>
      <td>
<strong>M.Sc. Biology</strong>, Ruhr University Bochum</td>
      <td>Â </td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Master Thesis: <em>Systematic Comparison of RNA Structure Probing Data.</em>
</td>
      <td>Â </td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2014 â€“ 2017</td>
      <td>
<strong>M.Sc. Biology</strong>, Martin Luther University Halle-Wittenberg</td>
      <td>Â </td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Specialization in Crops Science</td>
      <td>Â </td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2011 â€“ 2014</td>
      <td>
<strong>B.Sc. Biology</strong> , Leipzig University</td>
      <td>Â </td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Bachelor Thesis: <em>Study of the Nitrogen Acquisition of the Wheat Variety â€™Akteurâ€™.</em>
</td>
      <td>Â </td>
    </tr>
  </tbody>
</table>

<p><br></p>
<h3 id="publications">
<a class="anchor" href="#publications" aria-hidden="true"><span class="octicon octicon-link"></span></a>Publications</h3>
<hr>

<div class="publications">


<div class="unfancy_year">2025</div>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="article" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">A quartet-based approach for inferring phylogenetically informative features from genomic and phenomic data</div>
    <!-- Author -->
    <div class="author">
            <em>Brandenburg</em>,Â Hack,Â Mosig
    </div>

    
      <div class="periodical">
      
        <em>Computational and Structural Biotechnology Journal</em>
      
      
        Aug
      
      
        â€‰27
      
        
      
        2025
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.1016/j.csbj.2025.08.015" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Neural networks are widely used in bioinformatics to extract features from morphological, structural, and sequence data of different taxa. A key question is whether such features are compatible with a known phylogenetic tree describing the evolutionary relationships among the taxa. We address this question with a machine learning approach that takes taxon-specific data and a reference tree as input, and trains a neural network to produce a latent feature space whose pairwise distances are consistent with the tree topology. Our approach builds on the established role of quartets in distance-based phylogeny, leading to a quartet-based loss function for neural network training. In a proof-of-concept study using bacterial ribosomal RNA sequences, we show that the learned feature distances closely match the reference phylogeny. This framework can be applied to diverse biological data types, providing a principled way to incorporate phylogenetic constraints into neural network-based feature extraction.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="articlf" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Two temperature-responsive RNAs act in concert: the small RNA CyaR and the mRNA ompX</div>
    <!-- Author -->
    <div class="author">Guanzon,Â PienkoÃŸ,Â 
            <em>Brandenburg</em>,Â RÃ¶der,Â Scheller,Â Dietze,Â Wimbert,Â Twittenhoff,Â Narberhaus
    </div>

    
      <div class="periodical">
      
        <em>Nucleic acids research</em>
      
      
        Jan
      
      
        â€‰53
      
        
      
        2025
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.1093/nar/gkaf041" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Bacterial pathogens, such as Yersinia pseudotuberculosis, encounter temperature fluctuations during host infection and upon return to the environment. These temperature shifts impact RNA structures globally. While previous transcriptome-wide studies have focused on RNA thermometers in the 5â€™-untranslated region of virulence-related messenger RNAs, our investigation revealed temperature-driven structural rearrangements in the small RNA CyaR (cyclic AMP-activated RNA). At 25Â°C, CyaR primarily adopts a conformation that occludes its seed region, but transitions to a liberated state at 37Â°C. By RNA sequencing and in-line probing experiments, we identified the Shine-Dalgarno sequence of ompX as a direct target of CyaR. Interestingly, the ompX transcript itself exhibits RNA thermometer-like properties, facilitating CyaR base pairing at elevated temperatures. This interaction impedes ribosome binding to ompX and accelerates degradation of the ompX transcript. Furthermore, we observed induced proteolytic turnover of the OmpX protein at higher temperatures. Collectively, our study uncovered multilayered post-transcriptional mechanisms governing ompX expression, resulting in lower OmpX levels at 37Â°C compared with 25Â°C.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="lysr_schmidt_2025" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Two redox-responsive LysR-type transcription factors control the oxidative stress response of Agrobacterium tumefaciens</div>
    <!-- Author -->
    <div class="author">Schmidt,Â 
            <em>Brandenburg</em>,Â Elders,Â Shahzad,Â SchÃ¤kermann,Â Fiedler,Â Knoke,Â PfÃ¤nder,Â Dietze,Â Bille,Â GÃ¤rtner,Â Albin,Â Leichert,Â Bandow,Â Hofmann,Â Narberhaus
    </div>

    
      <div class="periodical">
      
        <em>Nucleic Acids Research</em>
      
      
        Apr
      
      
        â€‰53
      
        
        (6)
      
      
        2025
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.1093/nar/gkaf267" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Pathogenic bacteria often encounter fluctuating reactive oxygen species (ROS) levels, particularly during host infection, necessitating robust redox-sensing mechanisms for survival. The LysR-type transcriptional regulator (LTTR) OxyR is a widely conserved bacterial thiol-based redox sensor. However, members of the Rhizobiales also encode LsrB, a second LTTR with potential redox-sensing function. This study explores the roles of OxyR and LsrB in the plant-pathogen Agrobacterium tumefaciens. Through single and combined deletions, we observed increased H2O2 sensitivity, underscoring their function in oxidative defense. Genome-wide transcriptome profiling under H2O2 exposure revealed that OxyR and LsrB co-regulate key antioxidant genes, including katG, encoding a bifunctional catalase/peroxidase. Agrobacterium tumefaciens LsrB possesses four cysteine residues potentially involved in redox sensing. To elucidate the structural basis for redox-sensing, we applied single-particle cryo-EM (cryogenic electron microscopy) to experimentally confirm an AlphaFold model of LsrB, identifying two proximal cysteine pairs. In vitro thiol-trapping coupled with mass spectrometry confirmed reversible thiol modifications of all four residues, suggesting a functional role in redox regulation. Collectively, these findings reveal that A. tumefaciens employs two cysteine-based redox sensing transcription factors, OxyR and LsrB, to withstand oxidative stress encountered in host and soil environments.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li>
</ol>

<div class="unfancy_year">2024</div>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="lysr_schmidt" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">The LysR-type transcription factor LsrB regulates beta-lactam resistance in Agrobacterium tumefaciens</div>
    <!-- Author -->
    <div class="author">Schmidt,Â Remme,Â Eisfeld,Â 
            <em>Brandenburg</em>,Â Bille,Â Narberhaus
    </div>

    
      <div class="periodical">
      
        <em>Molecular Microbiology</em>
      
      
      
        â€‰121
      
        
        (1)
      
      
        2024
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/https://doi.org/10.1111/mmi.15191" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Agrobacterium tumefaciens is a plant pathogen, broadly known as the causal agent of the crown gall disease. The soil bacterium is naturally resistant to beta-lactam antibiotics by utilizing the inducible beta-lactamase AmpC. Our picture on the condition-dependent regulation of ampC expression is incomplete. A known regulator is AmpR controlling the transcription of ampC in response to unrecycled muropeptides as a signal for cell wall stress. In our study, we uncovered the global transcriptional regulator LsrB as a critical player acting upstream of AmpR. Deletion of lsrB led to severe ampicillin and penicillin sensitivity, which could be restored to wild-type levels by lsrB complementation. By transcriptome profiling via RNA-Seq and qRT-PCR and by electrophoretic mobility shift assays, we show that ampD coding for an anhydroamidase involved in peptidoglycan recycling is under direct negative control by LsrB. Controlling AmpD levels by the LysR-type regulator in turn impacts the cytoplasmic concentration of cell wall degradation products and thereby the AmpR-mediated regulation of ampC. Our results substantially expand the existing model of inducible beta-lactam resistance in A. tumefaciens by establishing LsrB as higher-level transcriptional regulator.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>

<div class="unfancy_year">2023</div>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="dis_brandenburg" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Investigation of functional RNA structures using deep learning and experiment-guided computational approaches</div>
    <!-- Author -->
    <div class="author">
           <em>Brandenburg</em>
    </div>

    
      <div class="periodical">
      
        <em>Doctoral Thesis, Ruhr University Bochum</em>
      
      
      
        
      
        2023
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.13154/294-10571" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p> RNA secondary structures fulfill diverse regulatory functions in the cell. Their investigation is therefore key to understanding biological processes. The Lead-Seq method established here makes an important contribution to this by enabling transcriptome-wide mapping of RNA secondary structures in vivo. Another method presented here complements the in silico prediction of RNA structures. In an inverse folding based pre-training approach, deep learning methods are enhanced with prior knowledge from the thermodynamic model. This enables the inference of features from only a few data points, as shown in the example of transcription terminators. Last, a novel quartet loss is introduced that continues the integration of existing knowledge in deep learning methods. The trained models reproduce a known phylogeny and can be used to identify which features evolve along the given tree.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>

<div class="unfancy_year">2022</div>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="Inverse_folding" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Inverse folding based pre-training for the reliable identification of intrinsic transcription terminators</div>
    <!-- Author -->
    <div class="author">
            <em>Brandenburg</em>,Â Narberhaus,Â Mosig
    </div>

    
      <div class="periodical">
      
        <em>PLOS Computational Biology</em>
      
      
      
        â€‰18
      
        
        (7)
      
      
        2022
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.1371/journal.pcbi.1010240" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>It is well-established that neural networks can predict or identify structural motifs of non-coding RNAs (ncRNAs). Yet, the neural network based identification of RNA structural motifs is limited by the availability of training data that are often insufficient for learning features of specific ncRNA families or structural motifs. Aiming to reliably identify intrinsic transcription terminators in bacteria, we introduce a novel pre-training approach that uses inverse folding to generate training data for predicting or identifying a specific family or structural motif of ncRNA. We assess the ability of neural networks to identify secondary structure by systematic in silico mutagenesis experiments. In a study to identify intrinsic transcription terminators as functionally well-understood RNA structural motifs, our inverse folding based pre-training approach significantly boosts the performance of neural network topologies, which outperform previous approaches to identify intrinsic transcription terminators. Inverse-folding based pre-training provides a simple, yet highly effective way to integrate the well-established thermodynamic energy model into deep neural networks for identifying ncRNA families or motifs. The pre-training technique is broadly applicable to a range of network topologies as well as different types of ncRNA families and motifs.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>

<div class="unfancy_year">2021</div>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="https://doi.org/10.1111/mmi.14695" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">A LysR-type transcriptional regulator controls the expression of numerous small RNAs in Agrobacterium tumefaciens</div>
    <!-- Author -->
    <div class="author">Eisfeld,Â Kraus,Â Ronge,Â Jagst,Â 
            <em>Brandenburg</em>,Â Narberhaus
    </div>

    
      <div class="periodical">
      
        <em>Molecular Microbiology</em>
      
      
      
        â€‰116
      
        
        (1)
      
      
        2021
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.1111/mmi.14695" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Small RNAs (sRNAs) are universal posttranscriptional regulators of gene expression and hundreds of sRNAs are frequently found in each and every bacterium. In order to coordinate cellular processes in response to ambient conditions, many sRNAs are differentially expressed. Here, we asked how these small regulators are regulated using Agrobacterium tumefaciens as a model system. Among the best-studied sRNAs in this plant pathogen are AbcR1 regulating numerous ABC transporters and PmaR, a regulator of peptidoglycan biosynthesis, motility, and ampicillin resistance. We report that the LysR-type regulator VtlR (also known as LsrB) controls expression of AbcR1 and PmaR. A vtlR/lsrB deletion strain showed growth defects, was sensitive to antibiotics and severely compromised in plant tumor formation. Transcriptome profiling by RNA-sequencing revealed more than 1,200 genes with altered expression in the mutant. Consistent with the function of VtlR/LsrB as regulator of AbcR1, many ABC transporter genes were affected. Interestingly, the transcription factor did not only control the expression of AbcR1 and PmaR. In the mutant, 102 sRNA genes were significantly up- or downregulated. Thus, our study uncovered VtlR/LsrB as the master regulator of numerous sRNAs. Thereby, the transcriptional regulator harnesses the regulatory power of sRNAs to orchestrate the expression of distinct sub-regulons.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>

<div class="unfancy_year">2020</div>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="10.1093/nar/gkaa404" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Lead-seq: transcriptome-wide structure probing in vivo using lead(II) ions</div>
    <!-- Author -->
    <div class="author">Twittenhoff*,Â 
            <em>Brandenburg*</em>,Â Righetti*,Â Nuss,Â Mosig,Â Dersch,Â Narberhaus
    </div>

    
      <div class="periodical">
      
        <em>Nucleic Acids Research</em>
      
      
      
        â€‰48
      
        
        (12)
      
      
        2020
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.1093/nar/gkaa404" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The dynamic conformation of RNA molecules within living cells is key to their function. Recent advances in probing the RNA structurome in vivo, including the use of SHAPE (Selective 2â€²-Hydroxyl Acylation analyzed by Primer Extension) or kethoxal reagents or DMS (dimethyl sulfate), provided unprecedented insights into the architecture of RNA molecules in the living cell. Here, we report the establishment of lead probing in a global RNA structuromics approach. In order to elucidate the transcriptome-wide RNA landscape in the enteric pathogen Yersinia pseudotuberculosis, we combined lead(II) acetate-mediated cleavage of single-stranded RNA regions with high-throughput sequencing. This new approach, termed â€˜Lead-seqâ€™, provides structural information independent of base identity. We show that the method recapitulates secondary structures of tRNAs, RNase P RNA, tmRNA, 16S rRNA and the rpsT 5â€²-untranslated region, and that it reveals global structural features of mRNAs. The application of Lead-seq to Y. pseudotuberculosis cells grown at two different temperatures unveiled the first temperature-responsive in vivo RNA structurome of a bacterial pathogen. The translation of candidate genes derived from this approach was confirmed to be temperature regulated. Overall, this study establishes Lead-seq as complementary approach to interrogate intracellular RNA structures on a global scale.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
    *shared first authorship
    
  </div>
</div>
</li></ol>


</div>

<p><br></p>

<h3 id="talks">
<a class="anchor" href="#talks" aria-hidden="true"><span class="octicon octicon-link"></span></a>Talks</h3>
<hr>
<div class="publications">


  <div class="unfancy_year">2024</div>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="gcb_2024" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Explainable phylogenetic features inferred with quartet-based deep learning models</div>
    <!-- Author -->
    <div class="author">German Conference on Bioinformatics
    </div>

    
      <div class="periodical">
      
        <em>Bielefeld</em>
      
      
      
        
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>

  <div class="unfancy_year">2023</div>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="talk_bled_2023" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">A quartet-based approach to infer phylogenetically informative characters from phenotypic data</div>
    <!-- Author -->
    <div class="author">Winter Seminar Theoretical Biochemistry Group (TBI), University of Vienna
    </div>

    
      <div class="periodical">
      
        <em>Bled, Slovenia</em>
      
      
      
        
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>

  <div class="unfancy_year">2021</div>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="talk_florida_2021" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">RNA structuromics: in vitro, in vivo and in silico</div>
    <!-- Author -->
    <div class="author">Conference Microbial RNA Regulation
    </div>

    
      <div class="periodical">
      
        <em>St. Petersburg, Florida, USA</em>
      
      
      
        
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="talk_minsk_2021" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Inverse-folding based pre-training for terminator detection</div>
    <!-- Author -->
    <div class="author">STUDATA: Students Conference on Recent Methods of ML and Data Analysis
    </div>

    
      <div class="periodical">
      
        <em>National Academy of Science, Minsk, Belarus</em>
      
      
      
        
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li>
</ol>

  <div class="unfancy_year">2020</div>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="talk_bled_2020" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Transcriptome-wide structure probing with Lead-Seq</div>
    <!-- Author -->
    <div class="author">Winter Seminar Theoretical Biochemistry Group (TBI), University of Vienna
    </div>

    
      <div class="periodical">
      
        <em>Bled, Slovenia</em>
      
      
      
        
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="talk_duesseldorf_2020" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Lead-seq: transcriptome-wide in vivo structure probing with lead</div>
    <!-- Author -->
    <div class="author">Seminar of the Research Group Synthetic Microbiology
    </div>

    
      <div class="periodical">
      
        <em>Heinrich-Heine-University DÃ¼sseldorf, Germany</em>
      
      
      
        
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li>
</ol>


</div>

<p><br></p>
<h3 id="my-subsection">
<a class="anchor" href="#my-subsection" aria-hidden="true"><span class="octicon octicon-link"></span></a>Posters</h3>
<hr>

<div class="publications">


<div class="unfancy_year">2023</div>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="poster_GCB_2023" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Inverse folding based pre-training for the reliable identification of intrinsic transcription terminators</div>
    <!-- Author -->
    <div class="author">German Conference on Bioinformatics
    </div>

    
      <div class="periodical">
      
        <em>Hamburg, Germany</em>
      
      
      
        
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="poster_ISMB_2023" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">A quartet-based approach for inferring phylogenetically informative features from genomic and phenomic data</div>
    <!-- Author -->
    <div class="author">Intelligent Systems for Molecular Biology/European Conference on Computational Biology
    </div>

    
      <div class="periodical">
      
        <em>Lyon, France</em>
      
      
      
        
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    
      
      <a href="/assets/pdf/poster_ISMB2023.pdf" class="btn btn-sm z-depth-0" role="button">Poster</a>
      
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li>
</ol>

<div class="unfancy_year">2021</div>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="poster_GCB_2021" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Prediction of Intrinsic Transcription Terminators using Deep Learning Methods</div>
    <!-- Author -->
    <div class="author">German Conference on Bioinformatics
    </div>

    
      <div class="periodical">
      
        <em>Halle (Saale), Germany</em>
      
      
      
        
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    
      
      <a href="/assets/pdf/poster_GCB_2021.pdf" class="btn btn-sm z-depth-0" role="button">Poster</a>
      
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>

<div class="unfancy_year">2020</div>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="poster_VAAM_2020" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Lead-seq: In vivo RNA structure probing on a transcriptome-wide scale</div>
    <!-- Author -->
    <div class="author">Conference of the Association for General and Applied Microbiology (VAAM)
    </div>

    
      <div class="periodical">
      
        <em>Leipzig, Germany</em>
      
      
      
        
      
      </div>
    

    <div class="links">
    
    
    
    
    
    
    
    
    
    
    
      
      <a href="/assets/pdf/poster_VAAM_2020.pdf" class="btn btn-sm z-depth-0" role="button">Poster</a>
      
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>


</div>

<p><br></p>
<h3 id="teaching">
<a class="anchor" href="#teaching" aria-hidden="true"><span class="octicon octicon-link"></span></a>Teaching</h3>
<hr>

<table>
  <thead>
    <tr>
      <th>â€ƒ    â€ƒ  â€ƒ  â€ƒ  â€ƒ  â€ƒ â€ƒ â€‰</th>
      <th>â€ƒ</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>2023 â€“ 2024</td>
      <td><strong>Big Data in Bioinformatics</strong></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Supervision of studentâ€™s projects</td>
    </tr>
    <tr>
      <td>Â </td>
      <td><em>Ruhr University Bochum, Bioinformatics</em></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2022 â€“ 2024</td>
      <td><strong>Literature Seminar Bioinformatics</strong></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Supervision of master students talks</td>
    </tr>
    <tr>
      <td>Â </td>
      <td><em>Ruhr University Bochum, Bioinformatics</em></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2019 â€“ 2024</td>
      <td><strong>Module Bioinformatics</strong></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Supervision of course days â€˜RNA structure predictionâ€™ &amp; â€˜Analysis of genome-wide RNA structure probing dataâ€™</td>
    </tr>
    <tr>
      <td>Â </td>
      <td><em>Ruhr University Bochum, Bioinformatics</em></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2019 â€“ 2023</td>
      <td><strong>Module Microbiology and Biotechnology</strong></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Supervision of course part â€˜Bioinformaticsâ€™</td>
    </tr>
    <tr>
      <td>Â </td>
      <td><em>Ruhr University Bochum, Microbial Biology</em></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2018 â€“ 2019</td>
      <td><strong>Database Systems</strong></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Evaluation of weekly exercises</td>
    </tr>
    <tr>
      <td>Â </td>
      <td><em>Ruhr University Bochum, Theoretical Computer Science</em></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
    </tr>
    <tr>
      <td>2014</td>
      <td><strong>Plant Physiology</strong></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Teaching assistance in practical course</td>
    </tr>
    <tr>
      <td>Â </td>
      <td><em>Leipzig University, Plant Physiology</em></td>
    </tr>
    <tr>
      <td>Â </td>
      <td>Â </td>
    </tr>
  </tbody>
</table>


          </article>

        </div>

    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        Â© Copyright 2025 Vivian Brandenburg. Last updated: September 19, 2025.
      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Mansory & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/mansory.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
  <script src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script src="/assets/js/common.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
  </body>
</html>

