<!DOCTYPE html>
<html lang="en">

  <!-- Head -->
  <head>        
    <!-- Metadata, OpenGraph and Schema.org -->
    

    <!-- Standard metadata -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <title>Vivian  Brandenburg | publications</title>
    <meta name="author" content="Vivian  Brandenburg" />
    <meta name="description" content="<!-- publications by categories in reversed chronological order. generated by jekyll-scholar. -->" />


    <!-- Bootstrap & MDB -->
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous" />

    <!-- Fonts & Icons -->
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@5.15.4/css/all.min.css" integrity="sha256-mUZM63G8m73Mcidfrv5E+Y61y7a12O5mW4ezU3bxqW4=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous">
    <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons">

    <!-- Code Syntax Highlighting -->
    <link rel="stylesheet" href="https://gitcdn.link/repo/jwarby/jekyll-pygments-themes/master/github.css" />

    <!-- Styles -->
    <link rel="icon" href="data:image/svg+xml,<svg xmlns=%22http://www.w3.org/2000/svg%22 viewBox=%220 0 100 100%22><text y=%22.9em%22 font-size=%2290%22>ðŸ§¬</text></svg>">
    <link rel="stylesheet" href="/assets/css/main.css">
    <link rel="canonical" href="https://vivianbrandenburg.github.io/publications/">

    <!-- Dark Mode -->
    <script src="/assets/js/theme.js"></script>
    <script src="/assets/js/dark_mode.js"></script><!-- google -->
    <meta name="google-site-verification" content="nPIY7yz_3yXwImUhSsPpRGhL2u4Fi5XV5S4Ot9bgtDI" />

  </head>

  <!-- Body -->
  <body class="fixed-top-nav">

    <!-- Header -->
    <header>

      <!-- Nav Bar -->
      <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top">
        <div class="container">
          <a class="navbar-brand title font-weight-lighter" href="https://vivianbrandenburg.github.io/"><b>Vivian</b> Brandenburg</a>
          <!-- Navbar Toggle -->
          <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar top-bar"></span>
            <span class="icon-bar middle-bar"></span>
            <span class="icon-bar bottom-bar"></span>
          </button>

          <div class="collapse navbar-collapse text-right" id="navbarNav">
            <ul class="navbar-nav ml-auto flex-nowrap">

              <!-- About -->
              <li class="nav-item ">
                <a class="nav-link" href="/">about</a>
              </li>
              

              <!-- Other pages -->
              <li class="nav-item dropdown ">
                <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">CV</a>
                <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
                  <a class="dropdown-item" href="/CV_academic/">academic CV</a>
                  <div class="dropdown-divider"></div>
                  <a class="dropdown-item" href="/assets/one_page_CV.pdf">one-page CV</a>
                </div>
              </li>
              <li class="nav-item ">
                <a class="nav-link" href="/projects/">projects</a>
              </li>
              <li class="nav-item active">
                <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a>
              </li>

              <!-- Toogle theme mode -->
              <div class="toggle-container">
                <a id="light-toggle">
                  <i class="fas fa-moon"></i>
                  <i class="fas fa-sun"></i>
                </a>
              </div>
            </ul>
          </div>
        </div>
      </nav>
    </header>

    <!-- Content -->
    <div class="container mt-5">
      <!-- page.html -->
        <div class="post">

          <header class="post-header">
            <h1 class="post-title">publications</h1>
            <p class="post-description"><!-- publications by categories in reversed chronological order. generated by jekyll-scholar. --></p>
          </header>

          <article>
            <div class="publications">


  <h2 class="year">2024</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="lysr_schmidt" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">The LysR-type transcription factor LsrB regulates beta-lactam resistance in Agrobacterium tumefaciens</div>
    <!-- Author -->
    <div class="author">Schmidt,Â Remme,Â Eisfeld,Â 
            <em>Brandenburg</em>,Â Bille,Â Narberhaus
    </div>

    
      <div class="periodical">
      
        <em>Molecular Microbiology</em>
      
      
    
        â€‰121(1)
      
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/https://doi.org/10.1111/mmi.15191" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Agrobacterium tumefaciens is a plant pathogen, broadly known as the causal agent of the crown gall disease. The soil bacterium is naturally resistant to beta-lactam antibiotics by utilizing the inducible beta-lactamase AmpC. Our picture on the condition-dependent regulation of ampC expression is incomplete. A known regulator is AmpR controlling the transcription of ampC in response to unrecycled muropeptides as a signal for cell wall stress. In our study, we uncovered the global transcriptional regulator LsrB as a critical player acting upstream of AmpR. Deletion of lsrB led to severe ampicillin and penicillin sensitivity, which could be restored to wild-type levels by lsrB complementation. By transcriptome profiling via RNA-Seq and qRT-PCR and by electrophoretic mobility shift assays, we show that ampD coding for an anhydroamidase involved in peptidoglycan recycling is under direct negative control by LsrB. Controlling AmpD levels by the LysR-type regulator in turn impacts the cytoplasmic concentration of cell wall degradation products and thereby the AmpR-mediated regulation of ampC. Our results substantially expand the existing model of inducible beta-lactam resistance in A. tumefaciens by establishing LsrB as higher-level transcriptional regulator.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>

  <h2 class="year">2023</h2>
  <ol class="bibliography">
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="quartet_learning" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">A quartet-based approach for inferring phylogenetically informative features from genomic and phenomic data</div>
    <!-- Author -->
    <div class="author">
            <em>Brandenburg</em>,Â Hack,Â Mosig
    </div>

    
      <div class="periodical">
      
        <em>published with Doctoral Thesis, Ruhr University Bochum</em>
      
      
    
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.13154/294-10571" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p> Evolutionary information is an important guide for uncovering biological mechanisms in structural biology as well as other fields of biology. Here, we present a neural network based approach that infers phylogenetically informative features in genotype or phenotype data by training a neural network in a way that the output features of the network match a presupposed reference phylogeny. Building on the well-established role of quartets in distance based phylogeny, our approach translates naturally to a quartet-based loss function for training neural networks. In a proof-of-concept study, we use quartet loss training to bacterial ribosomal RNA sequences and demonstrate that distances between the inferred features well reflect the reference phylogeny. Furthermore, we discuss how the approach can potentially be utilized to infer mechanisms in structural biology.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li>
<li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="dis_brandenburg" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Investigation of functional RNA structures using deep learning and experiment-guided computational approaches</div>
    <!-- Author -->
    <div class="author">
           <em>Brandenburg</em>
    </div>

    
      <div class="periodical">
      
        <em>Doctoral Thesis, Ruhr University Bochum</em>
      
      
    
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.13154/294-10571" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p> RNA secondary structures fulfill diverse regulatory functions in the cell. Their investigation is therefore key to understanding biological processes. The Lead-Seq method established here makes an important contribution to this by enabling transcriptome-wide mapping of RNA secondary structures in vivo. Another method presented here complements the in silico prediction of RNA structures. In an inverse folding based pre-training approach, deep learning methods are enhanced with prior knowledge from the thermodynamic model. This enables the inference of features from only a few data points, as shown in the example of transcription terminators. Last, a novel quartet loss is introduced that continues the integration of existing knowledge in deep learning methods. The trained models reproduce a known phylogeny and can be used to identify which features evolve along the given tree.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li>
</ol>

  <h2 class="year">2022</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="Inverse_folding" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Inverse folding based pre-training for the reliable identification of intrinsic transcription terminators</div>
    <!-- Author -->
    <div class="author">
            <em>Brandenburg</em>,Â Narberhaus,Â Mosig
    </div>

    
      <div class="periodical">
      
        <em>PLOS Computational Biology</em>
      
      
    
        â€‰18(7)
      
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.1371/journal.pcbi.1010240" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>It is well-established that neural networks can predict or identify structural motifs of non-coding RNAs (ncRNAs). Yet, the neural network based identification of RNA structural motifs is limited by the availability of training data that are often insufficient for learning features of specific ncRNA families or structural motifs. Aiming to reliably identify intrinsic transcription terminators in bacteria, we introduce a novel pre-training approach that uses inverse folding to generate training data for predicting or identifying a specific family or structural motif of ncRNA. We assess the ability of neural networks to identify secondary structure by systematic in silico mutagenesis experiments. In a study to identify intrinsic transcription terminators as functionally well-understood RNA structural motifs, our inverse folding based pre-training approach significantly boosts the performance of neural network topologies, which outperform previous approaches to identify intrinsic transcription terminators. Inverse-folding based pre-training provides a simple, yet highly effective way to integrate the well-established thermodynamic energy model into deep neural networks for identifying ncRNA families or motifs. The pre-training technique is broadly applicable to a range of network topologies as well as different types of ncRNA families and motifs.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>

  <h2 class="year">2021</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="https://doi.org/10.1111/mmi.14695" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">A LysR-type transcriptional regulator controls the expression of numerous small RNAs in Agrobacterium tumefaciens</div>
    <!-- Author -->
    <div class="author">Eisfeld,Â Kraus,Â Ronge,Â Jagst,Â 
            <em>Brandenburg</em>,Â Narberhaus
    </div>

    
      <div class="periodical">
      
        <em>Molecular Microbiology</em>
      
      
    
        â€‰116(1)
      
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.1111/mmi.14695" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Small RNAs (sRNAs) are universal posttranscriptional regulators of gene expression and hundreds of sRNAs are frequently found in each and every bacterium. In order to coordinate cellular processes in response to ambient conditions, many sRNAs are differentially expressed. Here, we asked how these small regulators are regulated using Agrobacterium tumefaciens as a model system. Among the best-studied sRNAs in this plant pathogen are AbcR1 regulating numerous ABC transporters and PmaR, a regulator of peptidoglycan biosynthesis, motility, and ampicillin resistance. We report that the LysR-type regulator VtlR (also known as LsrB) controls expression of AbcR1 and PmaR. A vtlR/lsrB deletion strain showed growth defects, was sensitive to antibiotics and severely compromised in plant tumor formation. Transcriptome profiling by RNA-sequencing revealed more than 1,200 genes with altered expression in the mutant. Consistent with the function of VtlR/LsrB as regulator of AbcR1, many ABC transporter genes were affected. Interestingly, the transcription factor did not only control the expression of AbcR1 and PmaR. In the mutant, 102 sRNA genes were significantly up- or downregulated. Thus, our study uncovered VtlR/LsrB as the master regulator of numerous sRNAs. Thereby, the transcriptional regulator harnesses the regulatory power of sRNAs to orchestrate the expression of distinct sub-regulons.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>

  <h2 class="year">2020</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="10.1093/nar/gkaa404" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Lead-seq: transcriptome-wide structure probing in vivo using lead(II) ions</div>
    <!-- Author -->
    <div class="author">Twittenhoff*,Â 
            <em>Brandenburg*</em>,Â Righetti*,Â Nuss,Â Mosig,Â Dersch,Â Narberhaus
    </div>

    
      <div class="periodical">
      
        <em>Nucleic Acids Research</em>
      
      
    
        â€‰48(12)
      
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
      <a href="https://doi.org/10.1093/nar/gkaa404" class="btn btn-sm z-depth-0" role="button" target="_blank" rel="noopener noreferrer">DOI</a>
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>The dynamic conformation of RNA molecules within living cells is key to their function. Recent advances in probing the RNA structurome in vivo, including the use of SHAPE (Selective 2â€²-Hydroxyl Acylation analyzed by Primer Extension) or kethoxal reagents or DMS (dimethyl sulfate), provided unprecedented insights into the architecture of RNA molecules in the living cell. Here, we report the establishment of lead probing in a global RNA structuromics approach. In order to elucidate the transcriptome-wide RNA landscape in the enteric pathogen Yersinia pseudotuberculosis, we combined lead(II) acetate-mediated cleavage of single-stranded RNA regions with high-throughput sequencing. This new approach, termed â€˜Lead-seqâ€™, provides structural information independent of base identity. We show that the method recapitulates secondary structures of tRNAs, RNase P RNA, tmRNA, 16S rRNA and the rpsT 5â€²-untranslated region, and that it reveals global structural features of mRNAs. The application of Lead-seq to Y. pseudotuberculosis cells grown at two different temperatures unveiled the first temperature-responsive in vivo RNA structurome of a bacterial pathogen. The translation of candidate genes derived from this approach was confirmed to be temperature regulated. Overall, this study establishes Lead-seq as complementary approach to interrogate intracellular RNA structures on a global scale.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
    *shared first authorship
    
  </div>
</div>
</li></ol>

  <h2 class="year">2019</h2>
  <ol class="bibliography"><li>
<!-- _layouts/bib.html -->
<div class="row">
  <div class="col-sm-2 abbr"></div>

  <!-- Entry bib key -->
  <div id="master_brandenburg" class="col-sm-8">
  
    <!-- Title -->
    <div class="title">Systematic Comparison of RNA Structure Probing Data</div>
    <!-- Author -->
    <div class="author">
           <em>Brandenburg</em>
    </div>

    
      <div class="periodical">
      
        <em>Master Thesis, Ruhr University Bochum</em>
      
      
    
      
      </div>
    

    <div class="links">
    
      <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a>
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    </div>

    <!-- Hidden abstract block -->
    
    <div class="abstract hidden">
      <p>Many bacterial pathogens are known to adjust their life style upon being exposed to diverse
environmental factors. Yersinia pseudotuberculosis, for example, regulates the transcription of
more than 300 genes depending on temperature indicating the ingestion by its mammalian
host. This adaptation is further complemented by several additional regulatory layers. RNA
thermometers (RNAT), for example, regulate the expression of the pathogenâ€™s major virulence
regulator LcrF on translational level, as well as the translation of several other genes. To
further investigate the influence of temperature on the structurome of Y. pseudotuberculosis in
vivo as well as in vitro, lead-seq and PARS (parallel analysis of RNA structure) experiments
were executed at both 25 Â°C and 37 Â°C. The chosen thermal conditions mimic the temperature
in the external reservoir as well as the hostâ€™s body temperature. This study aimed at the
analysis of the resulting data.
To generate information about pairing states from structure probing experiments, reactivity
scores were calculated from the resulting data. The reliability of information provided by
these scores was improved by implementing novel steps in the calculation pipeline,
accounting for lack of data as well as for the overall amplitude of background noise.
Temperature-induced unfolding of RNAs was found in vivo as well as in vitro. Furthermore, a
significant higher structuredness of untranslated regions at the 5Ê¹-end of the transcript (5Ê¹-
UTR) compared to the coding sequence was identified in vivo. This difference was not
detectable at 37 Â°C. A three-nucleotide structural pattern was found to be present in the coding
region of messenger RNAs (mRNAs) at both temperatures, but not in the non-coding regions.
The data resulting from in vivo structure probing were also used to improve RNA structure
prediction. The application of this method to RNAs sampled at both temperatures suggested
that mRNAs might be less thermo-stable than non-coding RNAs.
To identify structural changes standing out against the general unfolding of RNA, a new
method was developed. With this method, the accumulation of significant structure alteration
in the 5Ê¹-UTR of mRNAs in vivo was detected. The approach was applied to predict about 80
new RNAT candidates, among which at least six represent verified RNATs.</p>
    </div>
    

    <!-- Hidden bibtex block -->
    


    
  </div>
</div>
</li></ol>


</div>


          </article>

        </div>

    </div>

    <!-- Footer -->    
    <footer class="fixed-bottom">
      <div class="container mt-0">
        Â© Copyright 2024 Vivian Brandenburg. Last updated: May 18, 2024.
      </div>
    </footer>

    <!-- JavaScripts -->
    <!-- jQuery -->
  <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>

    <!-- Bootsrap & MDB scripts -->
  <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.2/dist/umd/popper.min.js" integrity="sha256-l/1pMF/+J4TThfgARS6KwWrk/egwuVvhRzfLAMQ6Ds4=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.min.js" integrity="sha256-SyTu6CwrfOhaznYZPoolVw2rxoY7lKYKQvqbtqN93HI=" crossorigin="anonymous"></script>
  <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script>

    <!-- Mansory & imagesLoaded -->
  <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script>
  <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script>
  <script defer src="/assets/js/mansory.js" type="text/javascript"></script>
    
  <!-- Medium Zoom JS -->
  <script src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.6/dist/medium-zoom.min.js" integrity="sha256-EdPgYcPk/IIrw7FYeuJQexva49pVRZNmt3LculEr7zM=" crossorigin="anonymous"></script>
  <script src="/assets/js/zoom.js"></script><!-- Load Common JS -->
  <script src="/assets/js/common.js"></script>

    <!-- MathJax -->
  <script type="text/javascript">
    window.MathJax = {
      tex: {
        tags: 'ams'
      }
    };
  </script>
  <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script>
  <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script>

    
  </body>
</html>

